CMakeLists.txt

```diff
@@ -83,7 +83,7 @@ endif()
 # IMPORTANT: To insure consistency please ensure the SAME of the first two
 # values are also assigned to the "VERSION" and "BUILD" variables in the native
 # qmake project file, which is NOW called: ./src/mudlet.pro
-set(APP_VERSION 4.10.1)
+set(APP_VERSION 4.11.1)
```

Comment:
Is `4.11.1` intended? Or should it be `4.11.0`

Reply:
Yeah mistake, localised to this PR.